### ISSUE TITLE
Update arg to mypy pre-commit checker

### ISSUE THREAD

**Issue by orSolocate** [PyCQA/pylint#5627]
### Bug description

Running pre-commit checkers on my environment shows `mypy` error related to `missing-imports` .

[link ](https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
)to mypy docs.

The error is gone when updating `.pre-commit-config.yaml` adding:

`args: [--ignore-missing-imports]`

Also in pre-commit `mirrors-mypy` [docs](https://github.com/pre-commit/mirrors-mypy) there's a recommendation to is this argument.

Should this be added to pylint too?

### Configuration

```ini
Python 3.8.10
```

### Command used

```shell
running pre-commit's script
```

### Pylint output

```shell
Cannot find implementation or library stub for module named "colorama" (https://mypy.readthedocs.io/en/stable/ files)

http://pylint/reporters/text.py:323:%20note:%20See%20https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```

### Expected behavior

mypy passed

### Pylint version

```shell
pylint 2.13.0-dev0
astroid 2.9.1
Python 3.8.10
```

### OS / Environment

Windows 10 Pro 64 bit

**Comment by DanielNoord** [comment]
I think we do this on a per-package basis in `setup.cfg`. @orSolocate Could you try and see if adding colorma there works for Windows? I think most currently active contributors work on Mac or Linux..

**Comment by orSolocate** [comment]
@DanielNoord  - I seem to have the same message again, and I see my fix in `setup.cfg` file was removed. do you have any idea where can I add the `ignore_missing_imports` attribute now?
